tiled/src/main/kotlin/ktx/tiled/tiledMap.kt

```diff
@@ -0,0 +1,99 @@
+package ktx.tiled
+
+import com.badlogic.gdx.maps.MapLayer
+import com.badlogic.gdx.maps.MapObject
+import com.badlogic.gdx.maps.MapProperties
+import com.badlogic.gdx.maps.tiled.TiledMap
+
+private val DEFAULT_MAP_LAYER = MapLayer().apply { name = "ktx-default-map-layer" }
+
+/**
+ * Extension method to directly access the [MapProperties] of a [TiledMap]. The type is automatically
+ * derived from the type of the given default value. If the property is not defined the defaultValue will be returned.
+ * @param key property name
+ * @param defaultValue default value in case the property is missing
+ * @return value of the property or defaultValue if property is missing
+ */
+inline fun <reified T> TiledMap.property(key: String, defaultValue: T): T = this.properties[key, defaultValue, T::class.java]
+
+/**
+ * Extension method to directly access the [MapProperties] of a [TiledMap]. If the property
+ * is not defined then this method returns null.
+ * @param key property name
+ * @return value of the property or null if the property is missing
+ */
+inline fun <reified T> TiledMap.propertyOrNull(key: String): T? = this.properties[key, T::class.java]
+
+/**
+ * Extension method to directly access the [MapProperties] of a [TiledMap] and its [containsKey][MapProperties.containsKey] method
+ * @param key property name
+ * @return true if the property exists. Otherwise false
+ */
+fun TiledMap.containsProperty(key: String) = properties.containsKey(key)
```

Comment:
This could also be an operator.

Reply:
Maybe a `contains` operator would work better to check for existing layers in case of a `TiledMap` though.

Reply:
refer to `MapProperties` extension proposal from `MapObject`